portfolio-project-matching-app/backend/daoUser.js

```diff
@@ -124,15 +146,22 @@ const getUserById = async (userId) => {
     */
     // get user doc snapshot and use to initialize user object
     const userSnap = await getDocSnapshotById('users', userId);
-    const user = User.fromDocSnapshot(userSnap.id, userSnap);
-
-    // get associated projects to populate user object's projects
-    user.projects = await getProjectsByUserId(user.id);
 
-    // get associated technologies to populate user object's technologies
-    user.technologies = await getTechnologiesByUserId(user.id);
+    // handle case where user does not exist
+    if (!userSnap.exists()) {
+        console.log(`invalid id: '${userId}' does not exist in 'users'`);
+        return -1;
```

Comment:
I should have had this here. Didn't think we would use this function much. Obviously, I was wrong.